templates/index.html

```diff
@@ -7,10 +7,19 @@
   <h1 class="u-off-screen">Snapcraft</h1>
   <div class="row">
     {% if nps %}
-      <div class="p-notification p-snapcraft-nps-banner">
-        <p class="p-notification__response">
-          Thank you so much 🙏. Your feedback is really valuable to us.
-        </p>
+      <div class="p-notification p-snapcraft-nps-banner js-notification-holder">
+        <div class="p-notification__response">
+          <p class="u-no-margin--bottom u-no-padding--top">
+            Thank you so much 🙏. Your feedback is really valuable to us. <a href="#" class="js-nps-comment-toggle">Provide further comments</a>
+          </p>
+          <div class="p-snapcraft-nps-banner__form js-nps-comment u-hide">
+            <form class="--dark">
```

Comment:
Why being so `--dark`?

Reply:
So the icon in the button is the light version... The suggested use is in a `p-strip--dark` but I don't want a script!

Reply:
Ah, the spinner. Would it work with the class on the button? And a comment in the template please, so we remember why it's there.